PIPs/PIP-12.md

```diff
@@ -49,7 +60,33 @@ A new genesis parameter is introduced `stateSyncConfirmationDelay` which  is sto
 
 #### Bor Consensus Rules
 
-Current Implementation:
+The existing implementation and proposed changes for calculating `from` and `to` are mentioned below (with few added abstractions from actual implementation for easy understanding):
```

Comment:
_with several abstractions added from the actual implementation for ease of understanding_